common/formatting/align.cc

```diff
@@ -94,6 +94,8 @@ struct AlignmentCell {
   // as a space-only column, usually no more than 1 space wide.
   int left_border_width = 0;
 
+  int TotalWidth() const { return left_border_width + compact_width; }
```

Comment:
I think this could be a static if it's used only as a helper here

Reply:
Adding static keyword causes an error when compiling.